website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -480,28 +487,29 @@ definition](/docs/connect/registration/service-registration) or
 
 - `envoy_bootstrap_json_tpl` - Specifies a template in Go template syntax that
   is used in place of [the default
-  template](https://github.com/hashicorp/consul/blob/b64bda880843afaaf44591c3200f921626716849/command/connect/envoy/bootstrap_tpl.go#L87)
+  template](https://github.com/hashicorp/consul/blob/71d45a34601423abdfc0a64d44c6a55cf88fa2fc/command/connect/envoy/bootstrap_tpl.go#L129)
```

Comment:
This sha is from `master` today, which is more current than whatever this was pinned to.